.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can we ignore adding .vs (Visual Studio, VSCode) files to the project? Maybe it's just my own nitpick but unsure if it adds any value.

Reply:
my bad. I was being lazy and use add . let me get rid of it

Reply:
@raygervais : new commit pushed without .vs files